nonsemantic/NonSemantic.DebugBreak.asciidoc

```diff
@@ -0,0 +1,93 @@
+SPIR-V NonSemantic DebugBreak Instructions
+===========================================
+
+Version 1.00
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors
+------------
+
+- Qingyuan Zheng, NVIDIA Corporation
+- Ashwin Lele, NVIDIA Corporation
+- Jeff Bolz, NVIDIA Corporation
+
+Notice
+------
+
+Copyright (c) 2019-2022 The Khronos Group Inc. Copyright terms at
+http://www.khronos.org/registry/speccopyright.html
+
+Status
+------
+
+- Not Yet Approved by the SPIR-V Working Group
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2022-08-18
+| Revision           | 1
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.6 Revision 2.
+
+This instruction set requires SPIR-V 1.0.
```

Comment:
Should this be 1.6 (which is the version that added non-semantic support to core)